spec/unit/puppet/type/rabbitmq_policy_spec.rb

```diff
@@ -171,13 +167,25 @@
     end.to raise_error(Puppet::Error, %r{Invalid ha-sync-batch-size value.*future})
   end
 
-  it 'accepts list value in ha-params when ha-mode = nodes' do
-    definition = { 'ha-mode' => 'nodes', 'ha-params' => ['rabbit@rabbit-01', 'rabbit@rabbit-02'] }
-    policy[:definition] = definition
-    expect(policy[:definition]['ha-mode']).to eq('nodes')
-    expect(policy[:definition]['ha-params']).to be_a(Array)
-    expect(policy[:definition]['ha-params'][0]).to eq('rabbit@rabbit-01')
-    expect(policy[:definition]['ha-params'][1]).to eq('rabbit@rabbit-02')
+  context 'accepts list value in ha-params when ha-mode = nodes' do
+    before do
+      policy[:definition] = definition
+    end
+
+    let(:definition) { { 'ha-mode' => 'nodes', 'ha-params' => ['rabbit@rabbit-01', 'rabbit@rabbit-02'] } }
+
+    it do
+      expect(policy[:definition]['ha-mode']).to eq('nodes')
+    end
```

Comment:
I might be tempted to use a single line for each of these?
ie.
```
it { expect(policy[:definition]['ha-mode']).to eq('nodes') }
```

Reply:
Will do. You think moving the outer bit to a context makes sense here though?

Reply:
Updated, btw, if you want to re-review